tools/st2ctl

```diff
@@ -51,6 +51,7 @@ function print_usage() {
     echo "usage: st2ctl {reload, clean}"
     echo "optional arguments:"
     echo "  --register-all                Register all."
+    echo "  --register-triggers           Register all triggers."
```

Comment:
Already updated `st2common/bin/st2ctl`. Its in this commit. Wasn't sure which was used, so I just updated both. (And I saw that your recent changes for --register-setup-virtualenvs were in both)

But yeah, seems like one should go bye bye. 👍